adapter/internal/discovery/xds/server.go

```diff
@@ -613,6 +625,11 @@ func GenerateHashedAPINameVersionIDWithoutVhost(name, version string) string {
 	return generateHashValue(name, version)
 }
 
+// GenerateIdentifierForAPIWithoutVersion generates an identifier unique to the API despite of the version
+func GenerateIdentifierForAPIWithoutVersion(vhost, name string) string {
```

Comment:
we can move it to utils if this is an util func. If not, we can make it a private func.

Reply:
Moved to a private function.